pubnub-kotlin/pubnub-kotlin-api/src/appleMain/kotlin/com/pubnub/api/PubNubImpl.kt

```diff
@@ -500,7 +500,7 @@ class PubNubImpl(private val pubNubObjC: KMPPubNub) : PubNub {
     override fun getChannelMetadata(channel: String, includeCustom: Boolean): GetChannelMetadata {
         return GetChannelMetadataImpl(
             pubnub = pubNubObjC,
-            metadataId = channel,
+            channel = channel,
```

Comment:
FYI
In comment you mentioned that you change it to channelId "I restored it to channelId like you suggested"

Reply:
Your comment became outdated after applying my first changes and GitHub didn't update it after subsequent changes. It's already `channelId` if you open [this file](https://github.com/pubnub/kotlin/blob/feat/custom-message-type-swift/pubnub-kotlin/pubnub-kotlin-api/src/appleMain/kotlin/com/pubnub/api/PubNubImpl.kt#L503) from my branch